\section{Methods}


\subsection{RNA Extraction, Library Preparation, and Sequencing}


We performed RNA extraction in TRIzol (Life Technologies, Grand Island, NY) according to manufacturer instructions, except with a higher concentration of glycogen as carrier (20 ng) and a higher relative volume of TRIzol to the expected material (1 mL, as in \cite{Lott:2011cc} and \cite{Combs:2013jy}). We quantified RNA concentrations using 


Libraries were quantified using a combination of Qubit High Sensitivity DNA (Life Technologies) and Bioanalyzer (Agilent Technologies, Sunnyvale, CA) readings, then pooled to equalize index concentration. Pooled libraries were then submitted to the Vincent Coates Genome Sequencing Laboratory for 50bp paired-end sequencing according to standard protocols for the Illumina HiSeq 2500. Bases were called using HiSeq Control Software v1.8 and Real Time Analysis v2.8.


\subsection{Mapping and Quantification}


Reads were mapped using STAR \cite{Dobin:2012fg} to a combination of the FlyBase reference genome version 5.54 for {\em D. melanogaster} and {\em D. virilis} \cite{McQuilton:2011iq}. We randomly sampled the mapped reads to use an equal number in each sample compared. We used {\tt htseq-count --idattr='gene_name' --stranded=no --sorted=pos} to count absolute read abundance per gene \cite{Anders_2014}.
