\section{Introduction}


In particular, we felt that published descriptions of ``single-cell'' and other low-volume protocols did not adequately address whether a change in concentration of a given RNA between two samples would result in a proportional change in the FPKM values between those samples. While there are biases inherent to any protocol, we were concerned that direct amplification of the mRNA would select for PCR compatible genes in difficult to predict, and potentially non-linear ways.  For certain applications, this may not be a critical flaw, for instance in assigning a single cell to a specific tissue type within a non-homogenous tissue.  However, to measure spatial and temporal activation of genes across an embryo, it is important that the output is monotonic with respect to concentration, and ideally linear.


While it is possible to estimate absolute numbers of cellular RNAs from an RNAseq experiment, doing so requires spike-ins of known concentration and estimates of total cellular RNA content \cite{Mortazavi:2008jj}. However, many RNAseq experiments do not do this controls, nor are such controls necessary if the absolute number is not necessary. While ultimately absolute concentrations will be necessary to fully predict properties such as noise tolerance of the regulatory circuits \cite{Gregor:2007du,Gregor:2005jn}, many current modeling efforts rely only on scaled concentration measurements, often derived from {\em in situ-hybridization} experiments \cite{Garcia:2013fs,Ilsley:2013fk,He:2010ix}.  Given that, we felt it was not important that different protocols should necessarily agree on the expression FPKM values for any given gene.
